api/app/signals/apps/api/filters/signal.py

```diff
@@ -334,12 +334,12 @@ class PublicSignalGeographyFilter(FilterSet):
 
     maincategory_slug = filters.ModelMultipleChoiceFilter(
         queryset=_get_parent_category_queryset(), to_field_name='slug',
-        field_name='category_assignment__category__parent__slug',
+        field_name='parent_category_slug',
```

Comment:
see comment on `api/app/signals/apps/signals/models/views/signal.py`

Reply:
The `field_name` has been removed, the `to_field_name` is still needed as it refers to the field on the model that is used to populate the `ModelMultipleChoiceFilter`. So the `slug` on the `Category` model in this case.